content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -1,17 +1,28 @@
 ---
-title: Optimization Improvements in DVC 1.0
+title: Remote Optimization Improvements in DVC 1.0
 date: 2020-06-29
 description: |
   An overview of how data synchronization to and from remote storage is optimized in DVC 1.0.
+picture: ''
 author: peter_rowlands
 ---
 
-One of the key features provided by DVC is the ability to efficiently sync
-versioned datasets between a user's local machine and
-[remote storage](https://dvc.org/doc/command-reference/remote), and version 1.0
-includes several performance optimizations related to synchronizing data with
-remotes. This post provides an overview of some of the ways in which remote
-performance has been optimized in DVC.
+One of the key features provided by DVC is the ability to efficiently
+synchronize versioned datasets between a user's local machine and
```

Comment:
Let's give a little more background about the problem here. Why are users so concerned with this optimization?  Could make the argument that noticeable data transfer times slow down iterations of a project, or it frustrates users. I also want to make clear that this is a general problem in ML projects, not strictly to do with DVC- could happen with rsync :)